russian/wardrobe-to-narnia/wardrobe.md

```diff
@@ -0,0 +1,6 @@
+Осторожно наступая на пол из глазури вы медленно, но уверенно двигаетесь к шкафу. 
+С детства знакомый шкаф может унести вас в страну, где пол из камня и твой лучший друг - говорящий лев. 
+Но чуда не произошло, то ли компот был слишком крепок, но в шкафу были толко пустые коробки от зефира, страрые футбольные бутсы Nike и 
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.